tests/snapshots/jit_spec__tests__executor_fragments_nested.snap

```diff
@@ -114,896 +114,6 @@ expression: data
-        "user": {
-          "id": 9,
-          "name": "Glenna Reichert",
-          "email": "Chaim_McDermott@dana.io",
-          "phone": "(775)976-6794 x41206",
-          "username": "Delphine"
-        }
-      },
-      {
-        "id": 90,
-        "user": {
-          "id": 9,
-          "name": "Glenna Reichert",
-          "email": "Chaim_McDermott@dana.io",
-          "phone": "(775)976-6794 x41206",
-          "username": "Delphine"
-        }
-      },
-      {
-        "id": 91,
-        "user": {
-          "id": 10,
-          "name": "Clementina DuBuque",
-          "email": "Rey.Padberg@karina.biz",
-          "phone": "024-648-3804",
-          "username": "Moriah.Stanton"
-        }
-      },
-      {
-        "id": 92,
-        "user": {
-          "id": 10,
-          "name": "Clementina DuBuque",
-          "email": "Rey.Padberg@karina.biz",
-          "phone": "024-648-3804",
-          "username": "Moriah.Stanton"
-        }
-      },
-      {
-        "id": 93,
-        "user": {
-          "id": 10,
-          "name": "Clementina DuBuque",
-          "email": "Rey.Padberg@karina.biz",
-          "phone": "024-648-3804",
-          "username": "Moriah.Stanton"
-        }
-      },
-      {
-        "id": 94,
-        "user": {
-          "id": 10,
-          "name": "Clementina DuBuque",
-          "email": "Rey.Padberg@karina.biz",
-          "phone": "024-648-3804",
-          "username": "Moriah.Stanton"
-        }
-      },
-      {
-        "id": 95,
-        "user": {
-          "id": 10,
-          "name": "Clementina DuBuque",
-          "email": "Rey.Padberg@karina.biz",
-          "phone": "024-648-3804",
-          "username": "Moriah.Stanton"
-        }
-      },
-      {
-        "id": 96,
-        "user": {
-          "id": 10,
-          "name": "Clementina DuBuque",
-          "email": "Rey.Padberg@karina.biz",
-          "phone": "024-648-3804",
-          "username": "Moriah.Stanton"
-        }
-      },
-      {
-        "id": 97,
-        "user": {
-          "id": 10,
-          "name": "Clementina DuBuque",
-          "email": "Rey.Padberg@karina.biz",
-          "phone": "024-648-3804",
-          "username": "Moriah.Stanton"
-        }
-      },
-      {
-        "id": 98,
-        "user": {
-          "id": 10,
-          "name": "Clementina DuBuque",
-          "email": "Rey.Padberg@karina.biz",
-          "phone": "024-648-3804",
-          "username": "Moriah.Stanton"
-        }
-      },
-      {
-        "id": 99,
-        "user": {
-          "id": 10,
-          "name": "Clementina DuBuque",
-          "email": "Rey.Padberg@karina.biz",
-          "phone": "024-648-3804",
-          "username": "Moriah.Stanton"
-        }
-      },
-      {
-        "id": 100,
-        "user": {
-          "id": 10,
-          "name": "Clementina DuBuque",
-          "email": "Rey.Padberg@karina.biz",
-          "phone": "024-648-3804",
-          "username": "Moriah.Stanton"
-        }
       }
```

Comment:
This test shouldn't have failed.

Reply:
this is not failed test - I just stripped number of entries with `_limit` as discussed above